homeassistant/components/switch/rachio.py

```diff
@@ -8,7 +8,7 @@
 from homeassistant.components.switch import SwitchDevice, PLATFORM_SCHEMA
 from homeassistant.const import CONF_ACCESS_TOKEN
 
-REQUIREMENTS = ['rachiopy==0.1.2']
+REQUIREMENTS = ['rachiopy==0.1.1']
```

Comment:
I just checked, and @rfverbruggen released his own update to it 10 days ago. It doesn't look like anything that would break this integration, but it'll need testing.